R/currencies.R

```diff
@@ -23,11 +23,10 @@ renminbi <- function(type = "dar") {
   )
 
 
-  if (missing(type) | type == "dar") {
-    type <- "dar"
+  if (type == "dar") {
     rmb_tibble <- get_bnm_tbl(paths[[type]])
     rmb_tibble[["term"]] <- names(rmb_tibble[["deposit"]])
-    spread(unnest(rmb_tibble, cols = c(.data$deposit)), "term", "deposit")
+    spread(unnest(rmb_tibble, cols = c(.data[["deposit"]])), "term", "deposit")

```

Comment:
given approximate matching, prefer `[[` in this project (although tibble does default to exact matching)